conductor_api/src/config.rs

```diff
@@ -559,6 +559,8 @@ pub struct AgentConfiguration {
     /// If set to true conductor will ignore keystore_file and instead use the remote signer
     /// accessible through signing_service_uri to request signatures.
     pub holo_remote_key: Option<bool>,
+    /// If true this agent will use dummy keys rather than a keystore file
+    pub test_agent: Option<bool>,
```

Comment:
```suggestion
    #[serde(default)]
    pub test_agent: bool,
```

Reply:
I'm kinda a fan of the `Option<bool>`. It has the added benefit of not serializing it if it is None so all the existing tests that test against hard coded TOML still work.

Reply:
I feel like that's a reason to make those hard coded TOML tests more lax! It's a huge PITA to add anything to the config.

Reply:
@willemolding  @maackle Instead of this boolean field, why can't we just have a test agent load an embedded key store file? This worked fine with lib3h when testing tls encryption, for instance.

Reply:
I would prefer @maackle's version without the `Option` here.
@struktured, I don't understand your suggstion. With a boolean field, how would out know what is a test agent?

Reply:
@lucksus I am suggesting we make (static?) agent configuration instances that point to embedded key stores.  Although maybe I missing something- is there a place that requires absolutely knowing you are running a test agent vs a normal agent?